packages/material-ui-lab/src/CalendarPicker/PickersCalendarHeader.tsx

```diff
@@ -140,9 +147,9 @@ function PickersCalendarHeader<TDate>(
 
   const utils = useUtils<TDate>();
 
-  const SwitchViewButton = components.SwitchViewButton || IconButton;
+  const SwitchViewButton = components.SwitchViewButton || PickersCalendarHeaderSwitchViewButton;
```

Comment:
I think this one is better because `PickersCalendarHeaderSwitchViewButton` already wrapped `IconButton`.

```js
<PickersCalendarHeaderSwitchViewButton
  as={components.SwitchViewButton}
>
```

Reply:
Yes, good point.